nixos/modules/services/misc/tzupdate.nix

```
@@ -28,7 +28,12 @@ in {
       wants = [ "network-online.target" ];
       after = [ "network-online.target" ];
       script = ''
-        timedatectl set-timezone "$(${lib.getExe pkgs.tzupdate} --print-only)"
+        set -uo pipefail
+        timezone="$(${lib.getExe pkgs.tzupdate} --print-only)"
+        if [[ ! -z "$timezone" ]]; then
+          echo setting timezone to "$timezone"
```

It would be nice if the log message used real sentence capitalization and punctuation, so that it's clear where the message starts and ends. It's also useful if variable values are in some way distinguished from the surrounding text. How about something like this?
```suggestion
          echo "Setting timezone to '$timezone'."
```

---

> It would be nice if the log message used real sentence capitalization and punctuation, so that it's clear where the message starts and ends. It's also useful if variable values are in some way distinguished from the surrounding text. How about something like this?

This is great, thanks!